pkg/quota/admission/clusterresourcequota/accessor.go

```diff
@@ -0,0 +1,161 @@
+package clusterresourcequota
+
+import (
+	"time"
+
+	lru "github.com/hashicorp/golang-lru"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kapierrors "k8s.io/kubernetes/pkg/api/errors"
+	utilquota "k8s.io/kubernetes/pkg/quota"
+	"k8s.io/kubernetes/pkg/storage/etcd"
+	utilwait "k8s.io/kubernetes/pkg/util/wait"
+
+	oclient "github.com/openshift/origin/pkg/client"
+	ocache "github.com/openshift/origin/pkg/client/cache"
+	quotaapi "github.com/openshift/origin/pkg/quota/api"
+	"github.com/openshift/origin/pkg/quota/controller/clusterquotamapping"
+)
+
+// QuotaAccessor abstracts the get/set logic from the rest of the Evaluator.  This could be a test stub, a straight passthrough,
+// or most commonly a series of deconflicting caches.
+type QuotaAccessor interface {
```

Comment:
Isn't this the same as what was added upstream already?

https://github.com/kubernetes/kubernetes/blob/master/plugin/pkg/admission/resourcequota/resource_access.go#L38?

Is this going to be removed after a rebase or something?